connections/evm/config.go

```diff
@@ -17,69 +18,71 @@ import (
 const DefaultGasLimit = 6721975
 const DefaultGasPrice = 20000000000
 
+var BlockRetryInterval = time.Second * 5
+
 // Config encapsulates all necessary parameters in ethereum compatible forms
 type Config struct {
```

Comment:
`Config` is relative to a chain, not a connection. It should remain in the `ethereum` package. To allow the connection to live in a separate package, the connection should no longer require that specific `Config`. Instead, we can define a config specifically for it.